src/vsr.zig

```diff
@@ -270,6 +270,9 @@ pub const BlockReference = struct {
 
 /// Viewstamped Replication protocol commands:
 pub const Command = enum(u8) {
+    // Looking to make backwards incompatible changes here? Make sure to check release.zig for
```

Comment:
I could make a comptime check that calculates the hash of all the fields and values - so if you update something, you have to update manually. Think it's worth it?

Reply:
I think it's _probably_ not worth it:

- It could provide a false sense of assurance -- not all upgrade-worthy changes would trigger the warning.
- (And also, it isn't necessarily required to upgrade in all such cases. For example, if you were adding a `command`, but it is safe for old versions to ignore the command.)